GUI2.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+dotnet run --project EDSEditorGUI2 --framework net8.0 --property WarningLevel=0 "$@"
```

Comment:
```suggestion
dotnet run --project EDSEditorGUI2 --property WarningLevel=0 "$@"
```

I did not need --framework, maybe only needed if project uses multiple .net versions

Reply:
https://github.com/CANopenNode/CANopenEditor/pull/197/commits/2b8d2d8a35d90867c4de5ac7600f847f3e743326